.github/workflows/avm.res.azure-stack-hci.virtual-machine-instance.yml

```
@@ -0,0 +1,88 @@
+name: "avm.res.azure-stack-hci.virtual-machine-instance"
+
+on:
+  workflow_dispatch:
+    inputs:
+      staticValidation:
+        type: boolean
+        description: "Execute static validation"
+        required: false
+        default: true
+      deploymentValidation:
+        type: boolean
+        description: "Execute deployment validation"
+        required: false
+        default: true
+      removeDeployment:
+        type: boolean
+        description: "Remove deployed module"
+        required: false
+        default: true
+      customLocation:
+        type: string
+        description: "Default location overwrite (e.g., eastus)"
+        required: false
+  push:
+    branches:
+      - main
+    paths:
+      - ".github/actions/templates/avm-**"
+      - ".github/workflows/avm.template.module.yml"
+      - ".github/workflows/avm.res.azure-stack-hci.virtual-machine-instance.yml"
+      - "avm/res/azure-stack-hci/virtual-machine-instance/**"
+      - "avm/utilities/pipelines/**"
+      - "!avm/utilities/pipelines/platform/**"
```

Please update to current path
```suggestion
      - "utilities/pipelines/**"
      - "!utilities/pipelines/platform/**"
```